CONTRIBUTION.md

```diff
@@ -31,18 +32,21 @@ GIT CLONE: https://github.com/PatrykKuniczak/memesChat.git
         START FRONTEND: 'npm start'
 
 FOR DOCKER
-    The hints for env values it's the same as in standalone envs config
+
+The hints for env values it's the same as in standalone envs config
+
```

Comment:
`values are the same`, not `values it's the same`

Reply:
done